Movies description page

diogent opened the pull request:
link to the User Story: https://trello.com/c/49fMIb1H

Commit:
New project has been created)

Commit:
Sign up from has been created

Commit:
Added Login page and main page.

Commit:
Just renamed some <li>

Commit:
Bugs are fixed. DI was integrated.

Commit:
Added navbar for "Sign in" and "Sign Up" links

Commit:
no message

Commit:
Added ApplicationDbMovies class library

Commit:
Data -> Business Logic -> UI integrated. Database structure has been created.

Commit:
Added Database structure and init

Commit:
sedeed data. Created Home controller

Commit:
Main page UI was created

Commit:
Added Movies Description page.

Commit:
Added picture url's from database.

StanislauTarazevich commented on CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Queries/DbService.cs:
```diff
@@ -0,0 +1,45 @@
+
+namespace CinemaTicketSalesBusinessLogic.Queries
+{
+    public static class DbService
```
This should not be static. You should inject it into other modules with autofac

StanislauTarazevich commented on CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Queries/DbService.cs:
```diff
@@ -0,0 +1,45 @@
+{
+    public static class DbService
+    {
+        public static ApplicationDbContext db = new ApplicationDbContext();
```
You shouldn't create a context here, it should be injected.

StanislauTarazevich commented on CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Queries/DbService.cs:
```diff
@@ -0,0 +1,45 @@
+        /// <summary>
+        /// Using in HomeController for viewing Movies collection on Main page.
+        /// </summary>        
+        public static IEnumerable<MovieViewModel> GetMovieInfo()
```
MovieViewModel shouldn't be returned here.
In 3-layer architecture you should have models for each leayer.

For example:
DAL - Movie
BL - MovieDto
Web - MovieViewModel

And you should return BL model from BL layer method.

StanislauTarazevich commented on CinemaTicketSalesSystem/CinemaTicketSalesBusinessLogic/Queries/DbService.cs:
```diff
@@ -0,0 +1,45 @@
+        public static IEnumerable<MovieViewModel> GetMovieInfo()
+        {
+
+            return (from s in db.Movies
```
What are the advantages of query syntax vs method syntax?

StanislauTarazevich commented on CinemaTicketSalesSystem/CinemaTicketSalesSystem/Controllers/HomeController.cs:
```diff
@@ -10,14 +11,24 @@ namespace CinemaTicketSalesSystem.Controllers
     public class HomeController : Controller
     {
-        ApplicationDbContext db = new ApplicationDbContext();
+        private ApplicationDbContext db = new ApplicationDbContext();
```
Usually private keyword is not used. Just name your variable with '_' in the beginning, it is commonly used practice for private variables.
```
ApplicationDbContext _db = new ApplicationDbContext();
```

Commit:
Minor bug fix

Commit:
More Info on the description page

Commit:
description page design has been created

Commit:
Fixed all the stuff that mentioned in comments:
https://github.com/diogent/Cinema-Ticket-Sales-System/pull/1
https://github.com/diogent/Cinema-Ticket-Sales-System/pull/2
https://github.com/diogent/Cinema-Ticket-Sales-System/pull/3
